public/index.html

```diff
@@ -13,7 +13,7 @@
     </script>
   </head>
 
-  <body class="cpd-theme-dark">
+  <body class="nodisplay">
```

Comment:
Why do we need to hide the UI from within Element Call? Doesn't Element Web now hide the widget until the content is loaded?

Reply:
Yes this is somewhat redundent. It is only used for the SPA where we dont have a good way to change the theme.
But it might come in handy once we change it to use the system theme for instance. Then we can have is as nodisplay (browser doing some default background. Which should also conform to system theme) and once js has started evaluating what theme is actually used we use the correct color from the start.